internal/envoy/v3/listener.go

```diff
@@ -524,6 +524,17 @@ func SocketAddress(address string, port int) *envoy_core_v3.Address {
 			},
 		}
 	}
+	// Check if the address is a socket.
+	if strings.Contains(address, "/") {
```

Comment:
could use https://pkg.go.dev/os#PathSeparator here though this could get interesting if we had a socket address that the contour controller (not bootstrap) was programming somehow for a non-unix host